packages/internet-header/src/components/post-internet-header/post-internet-header.scss

```diff
@@ -108,6 +109,23 @@
 
   & > * {
     border-left: 1px solid color.$gray-10;
+    flex: 0 0 auto;
+  }
+}
+
+.main-navigation-custom-content {
+  max-height: var(--header-height);
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  flex-shrink: 1;
+
+  ::slotted(*) {
+    padding-inline: 1rem;
+
+    @include mixins.min(lg) {
+      padding-inline: 1.125rem;
+    }
```

Comment:
I'm not sure that this is a good idea. The responsibility for the slot is by the project. If somebody puts two sibling elements into the slot, both will have the padding. That might not be desired.

A better approach would be to document what we use for padding, so people know how to re-create the look in their custom integration.

Reply:
I added this default padding because 1.125rem (18px) is used for the KLP login button but not reproducible with our utility classes whereas `.mx-0` is. 

Just removed it but should we document this exact 1.125rem value, update the KLP button or create a class specific to the header buttons?

Reply:
I think we'd need to update the button padding. Kaloyan documented the component in Figma and used a different padding and that one looks better IMO. But that can be handled in a separate ticket. Documenting the padding for others would be a good idea.

Reply:
I created the ticket #2390